File: 프로그래머스/1/388351. 유연근무제/유연근무제.py
def solution(schedules, timelogs, startday):
    answer = 0
    
    for i in range(len(schedules)):
        time = schedules[i]//100*100 + ((schedules[i]+10)%100)//60*100 + ((schedules[i]+10)%100)%60
        print(((schedules[i]+10)%100)%60)
        day = startday-1
        
        answer += 1
        for j in timelogs[i]:
            if(day < 5 and j > time):
                answer -= 1
                break
            day = (day+1)%7
        
    return answer

File: 프로그래머스/1/388351. 유연근무제/유연근무제.java
class Solution {
    public int solution(int[] schedules, int[][] timelogs, int startday) {
        int answer = 0;
        
        for(int i=0; i<schedules.length; i++){
            int time = ((schedules[i]+10)/100 + ((schedules[i]+10)%100)/60)*100 // 시
                       + ((schedules[i]+10)%100)%60; // 분
            
            int cnt = 0;
            for(int j=0; j<timelogs[0].length; j++){
                int log = timelogs[i][j];
                boolean isWeek = ((startday+j)%7 > 0 && (startday+j)%7 < 6);
                
                if(isWeek && log <= time){
                    cnt += 1;
                }
            }
            if(cnt >= 5){
                answer += 1;
            }
            
        }
        
        
        return answer;
    }
}